packages/pluginutils/types/index.d.ts

```diff
@@ -11,7 +11,7 @@ export interface AttachedScope {
 export interface DataToEsmOptions {
   compact?: boolean;
   indent?: string;
-  namedExports?: boolean;
+  namedExports?: boolean | 'include-arbitrary-name';
```

Comment:
```suggestion
  namedExports?: boolean | 'include-arbitrary-name';
```

personally not a fan of mixing types on options. I'd rather see a `includeAtrbitraryNames: boolean` that worked in conjunction with `namedExports`. This increases the footprint of `DataToEsmOtions` but clearly defines behavior.

Reply:
Sure. I changed to that.